lib/config-generator.js

```diff
@@ -413,6 +413,10 @@ class ConfigGenerator {
         const outputPath = this.webpackConfig.outputPath.replace(this.webpackConfig.getContext() + '/', '');
         plugins.push(new AssetsOutputDisplayPlugin(outputPath, friendlyErrorsPlugin));
 
+        if (this.webpackConfig.useDevServer()) {
```

Comment:
It may not make any difference, but should we also use `this.webpackConfig.useHotModuleReplacementPlugin()`? for this `if` statement as well?